Add missing "properties" list.
--- libgudev-231/tests/test-enumerator-filter.c.orig	2017-02-19 12:08:49.443366196 +0100
+++ libgudev-231/tests/test-enumerator-filter.c	2017-02-19 12:09:00.573366068 +0100
@@ -47,6 +47,7 @@
 				     "ID_MODEL", "KoolGadget 2", NULL);
 
 	umockdev_testbed_add_device (testbed, "drm", "dev3", NULL,
+			             NULL,
 				     "ID_FOR_SEAT", "drm-pci-0000_00_02_0", NULL);
 
 	/* Check the number of items in GUdevClient */
